api/tests/integration/infrastructure/repositories/schooling-registration-repository_test.js

```diff
@@ -1436,9 +1436,11 @@ describe('Integration | Infrastructure | Repository | schooling-registration-rep
 
   describe('#getSchoolingRegistrationInformationByNationalStudentIdFirstNameLastNameAndBirthdate', () => {
 
-    it('should return schooling registration information', async () => {
+    it('should return the latest schooling registration information', async () => {
```

Comment:
**suggestion (chipotage)**
Ce test documente 2 choses différentes si j'ai bien compris : 
1- il doit renvoyer les bonnes infos 
2- il doit choisir le schooling reg le plus à jour (celui avec l'orga la plus récente)
Peut être qu'on aurait pu faire 2 `it` distinct pour insister sur le fait qu'on choisit bien le dernier schooling-reg ?

Reply:
done